sos/report/plugins/fibrechannel.py

```diff
@@ -18,8 +18,15 @@ class Fibrechannel(Plugin, RedHatPlugin):
     plugin_name = 'fibrechannel'
     profiles = ('hardware', 'storage', 'system')
     files = ('/sys/class/fc_host', '/sys/class/fc_remote_ports')
+    option_list = [("add_path", "additional path to collect", "fast", None)]
 
     def setup(self):
         self.add_blockdev_cmd("udevadm info -a %(dev)s", devices='fibre')
+        additional_path = self.get_option("add_path")
+        if additional_path is not None:
+            path_list = additional_path.split(',')
+            if path_list:
+                for path in path_list:
```

Comment:
We can tighten this up a bit:

```python
for path in self.get_option('add_path').split(':'):
    self.add_copy_spec(path)
```

`add_copy_spec` has the logic to skip non existing paths and the like, so no need to put that into plugins.

Reply:
Thank you a lot! I've fixed as you wrote.

Reply:
Sorry, I haven't noticed the first comment. I'll fix it a bit later.